ui/js/models/perf/alerts.js

```diff
@@ -93,21 +98,23 @@ treeherder.factory('PhAlerts', [
                 if (copySummary) {
                     resultStr += "\n";
                 }
-                resultStr += "Summary of tests that regressed:\n\n" +
+                resultStr += "== Regressions ==\n\n" +
```

Comment:
I would just write this as "Regressions:". Same below with "== Improvements ==" (just "Improvements:" is fine)